next-frontend/src/lib/i18n/get18n.ts

```diff
@@ -0,0 +1,19 @@
+import i18next from "./i18n";
+// import { headerName } from "./settings";
+// import { headers } from "next/headers";
+
+export async function getT(options?: { keyPrefix?: string }) {
+  // const headerList = await headers();
+  const lng = "en";
+  if (lng && i18next.resolvedLanguage !== lng) {
+    await i18next.changeLanguage(lng);
+  }
+  return {
+    t: i18next.getFixedT(
+      lng ?? i18next.resolvedLanguage,
```

Comment:
Not sure how `lng` can ever be `undefined` if you're hard-setting it to `en` above. So I think this `??` will never ever trigger...